### PR TITLE
displays: Add X11 scaling settings

### PR DESCRIPTION
**Pull request by Drakulix**
~~Depends on https://github.com/pop-os/cosmic-comp/pull/779 being merged first.~~ cosmic-comp changes merged.

### CODE REVIEW

File: cosmic-settings/src/pages/display/mod.rs [hunk 1]
```diff
@@ -1098,6 +1127,43 @@ pub fn display_configuration() -> Section<crate::pages::Message> {
...
+    Section::default()
+        .title(fl!("legacy-applications"))
+        .descriptions(descriptions)
+        .view::<Page>(move |_binder, page, section| {
+            let descriptions = &section.descriptions;
+            widget::settings::view_section(&section.title)
+                .add(widget::settings::item_row(vec![widget::radio(
+                    widget::column()
+                        .push(text::heading(&descriptions[system]))
+                        .push(text(&descriptions[system_desc])),
```
Comment (git-f0x) [inline cosmic-settings/src/pages/display/mod.rs:1147]: Looking at the designs, this should probably be `text::body` and `text::caption` for system and system_desc (and native/native_desc), respectively.

Comment (Drakulix) [reply cosmic-settings/src/pages/display/mod.rs]: Totally right, thanks for pointing that out.

**Comment by git-f0x** [comment]
I'm on my phone currently, so I can't do a proper comment, but line 1088 should use `space_xs`, and the `.push(...)` at line 1123 should be moved to an `if page.list.outputs.len() == 1` just above, in order to close #528.

**Comment by Drakulix** [comment]
https://github.com/pop-os/cosmic-settings/issues/528 should be addressed by the latest changes.

**Review by mmstick — APPROVED** [review]

**Pull request merged**